service/src/test/java/tds/assessment/services/impl/AssessmentAssemblerTest.java

```diff
@@ -19,1018 +17,99 @@
 import tds.assessment.Strand;
 import tds.common.Algorithm;
 
-import static io.github.benas.randombeans.api.EnhancedRandom.randomListOf;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class AssessmentAssemblerTest {
     @Test
     public void shouldAssembleASegmentedFixedFormAssessment() {
-        // Assessment Init
-        final String assessmentId = "foo";
         Assessment assessment = new Assessment();
-        assessment.setKey("theKey");
```

Comment:
These tests were hard to follow, and I struggled with modifying them after the logic changes.  After some analysis it seems like a lot of the data created for these tests doesn't actually check a unique logic block. 

Instead of modifying all the data I created test data that would hit all the different logic blocks simplifying it for future maintainers.